py3status/modules/mpris.py

```diff
@@ -278,6 +279,10 @@ def metadata(self, metadata=None):
 
             self._metadata["length"] = self._get_time_str(metadata.get(Metadata_Map.LENGTH))
 
+            # we are converting the attribute name to lowercase because although the spec 
+            # says it's `xesam:trackNumber`, VLC exposes it as `xesam:tracknumber`
+            self._metadata["trackno"] = metadata.get(Metadata_Map.TRACK_NUMBER.lower());
```

Comment:
Placeholder should be `{tracknumber}` imho.

Reply:
Thanks @lasers - I've added commit c6a8206 which changes this.